### DISCUSSION TITLE
USB MIDI Adapter and pitch bend

### DISCUSSION THREAD

**Discussion by ZazaQumelashvili** [tttapa/Control-Surface#992] [Q&A]
I made a USB MIDI Adapter from the source: **https://www.hackster.io/dennis-v/usb-midi-adapter-e6ee57**.
  thanks, everything works, but I need to add a pitch bend rezistor to it. I don’t know how to program.. please add  to this code a pitch bend resistor to the A0 input. the voltage varies from 0 to 5 volts. central position from 2.2 to 2.8 volts. Arduino pro micro.
**Thank you!**

**Comment by tttapa** [comment]
See <https://tttapa.github.io/Control-Surface-doc/Doxygen/d3/d4e/Pitch-Bend-Potentiometer_8ino-example.html>.

**Reply by ZazaQumelashvili** [reply]
Thanks, but how can I combine this sketch with the sketch in this article?  https://www.hackster.io/dennis-v/usb-midi-adapter-e6ee57
I tried it and it didn't work

**Reply by tttapa** [reply]
Please post your attempt, including any error messages. Explain the desired behavior, and how that differs from the actual behavior you're seeing.

**Reply by ZazaQumelashvili** [reply]
everything is very simple: there are two sketches: **USB MIDI Adapter** and **pitch bend**. both use the same library the sketches are small, simple. I want to combine them in one, but I don’t know how.
please help with this
